client/my-sites/promote-post-i2/components/post-item/index.tsx

```diff
@@ -50,6 +50,8 @@ export default function PostItem( { post }: Props ) {
 	const likeCount = post?.like_count ?? 0;
 	const commentCount = post?.comment_count ?? 0;
```

Comment:
Not related to this PR, but I don't know if those numbers are formatted. Maybe we can check and format them so they have a thousand separators.

Reply:
That's awesome suggestion, Sebastian. Hope to add it in a following PR.